Mixpanel/MPDisplayTrigger.m

```diff
@@ -1,7 +1,7 @@
 #import "MPLogger.h"
 #import "MPDisplayTrigger.h"
+#import "SelectorEvaluator.h"
 #import "Mixpanel.h"
-#import "Mixpanel/Mixpanel-Swift.h"
 
 NSString * const ANY_EVENT = @"$any_event";
```

Comment:
```suggestion
static NSString *const ANY_EVENT = @"$any_event";
```
static is preferred, since it means the const is specific and internal to this .m file only